clang/test/CodeGen/aarch64-sme-func-attrs-inline/sme-inline-streaming-compatible-caller.c

```diff
@@ -0,0 +1,7 @@
+// RUN: %clang --target=aarch64-none-linux-gnu -march=armv9-a+sme -O3 -S -Xclang -verify %s
```

Comment:
You can remove `-O3` because clang doesn't do any codegen when it has `-verify`.

Reply:
Done.